api/go/api_polls_service.go

```diff
@@ -104,7 +108,27 @@ func (s *PollsApiService) GetPollResults(ctx context.Context, pollID int32) (Imp
 	//TODO: Uncomment the next line to return response Response(404, {}) or use other options such as http.Ok ...
 	//return Response(404, nil),nil
 
-	return Response(http.StatusNotImplemented, nil), errors.New("GetPollResults method not implemented")
+	// EXAMPLE!!
+	// TODO: Remove when actual implementation happens?
+	cntx := context.Background()
+	client := CreateFirestoreClient(cntx)
+	iter := client.Collection("binams-test-collection").Documents(ctx)
+
+	all_stuff := "This is gathered from firestore binams-test-collection:\n"
+
+	for {
+		doc, err := iter.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			log.Fatalf("Failed to iterate: %v", err)
+		}
+		data, _ := json.Marshal(doc.Data())
+		all_stuff += string(data) + "\n"
+	}
```

Comment:
If we need to do this for each collection (example if we need to do this for Polls and also for Votes) then we might want to move this into the firestore_client.go to dry it up.